pkg/remote/codec/unknown/unknown.go

```diff
@@ -0,0 +1,248 @@
+package unknown
+
+import (
+	"context"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	thrif "github.com/apache/thrift/lib/go/thrift"
+	"github.com/cloudwego/dynamicgo/thrift"
+	"github.com/cloudwego/kitex/pkg/protocol/bthrift"
+	"github.com/cloudwego/kitex/pkg/remote"
+	"github.com/cloudwego/kitex/pkg/remote/codec"
+	"github.com/cloudwego/kitex/pkg/remote/codec/perrors"
+	unknowns "github.com/cloudwego/kitex/pkg/remote/codec/unknown/service"
+	"github.com/cloudwego/kitex/pkg/rpcinfo"
+	"github.com/cloudwego/kitex/pkg/serviceinfo"
+)
+
+// UnknownCodec implements PayloadCodec
+type unknownCodec struct {
+	Codec remote.PayloadCodec
+}
+
+func NewUnknownCodec(code remote.PayloadCodec) remote.PayloadCodec {
+	return &unknownCodec{code}
+}
+
+func (c unknownCodec) Marshal(ctx context.Context, msg remote.Message, out remote.ByteBuffer) error {
+	nw, _ := out.(remote.NocopyWrite)
+	ink := msg.RPCInfo().Invocation()
+	data := msg.Data()
+	msgType := msg.MessageType()
+	if data, ok := data.(*unknowns.Result); ok {
+		//svcInfo := message.ServiceInfo()
+		//unknowns.SetServiceInfo(svcInfo, message.ProtocolInfo().CodecType, data.Method, data.Method)
```

Comment:
Please delete unnecessary codes, not just commented out.

Reply:
okay